black/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7-alpine
+MAINTAINER Pat Brisbin <pbrisbin@gmail.com>
```

Comment:
If you're willing to support these going forward, it'd be great to list yourself here IMO. I would also consider having a key under `metadata` in `info` to hold that too, though I can't think of any automation to hang off it just yet.

Reply:
yeah I can help out here -- of note the `MAINTAINER` instruction in docker is [actually deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated) so I left this alone

Reply:
Oh I didn't know that. Please go ahead and use `LABEL` for any `Dockerfiles` you add here and I'll make a change across the others to that later.

Reply:
fixed!